zhaquirks/lixee/zlinky.py

```diff
@@ -186,3 +195,19 @@ class ZLinkyTIC(CustomDevice):
             },
         },
     }
+
+
+class ZLinkyTICFWV12(ZLinkyTIC):
+    """ZLinky_TIC from LiXee with firmware v12.0+."""
+
+    signature = deepcopy(ZLinkyTIC.signature)
+
+    # Update model info
+    signature[MODELS_INFO] = [(LIXEE, "ZLinky_TIC_FW_V12")]
+
+    # Insert PowerConfiguration cluster in signature for devices with firmware v12.0+
+    signature[ENDPOINTS][1][INPUT_CLUSTERS] = (
+        signature[ENDPOINTS][1][INPUT_CLUSTERS][:1]
+        + [PowerConfiguration.cluster_id]
+        + signature[ENDPOINTS][1][INPUT_CLUSTERS][1:]
```

Comment:
Just a comment, tt is not necessary that the definition of the clusters be ordered.

Reply:
Thanks, I was wondering if it was necessary and/or cleaner to insert it.

Can switch to just appending `PowerConfiguration.cluster_id` to `signature[ENDPOINTS][1][INPUT_CLUSTERS]` if preferred.

Reply:
Just a matter of taste.
I'm not skilled in Python (but getting better thanks to HA!) so I had to make an effort to understand what was going on, once understood is fine to me.

Reply:
Now that you mention it, using `list.insert(i, x)` is probably more readable.

https://docs.python.org/3/tutorial/datastructures.html

Reply:
Simplified code with `list.insert(i, x)`